src/components/LogIn.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { Container, Card, Image, Button, Icon } from 'semantic-ui-react';
 import Stage from '../images/stage.jpg';
 import Logo from '../images/logo.svg';
+import Forward from '../images/forward.png';
```

Comment:
consistent image types, Forward should be converted to svg in Sketch. stage is a large image so that can stay a jpeg.